src/Php84/Php84.php

```diff
@@ -63,4 +63,48 @@ public static function mb_lcfirst(string $string, ?string $encoding = null): str
 
         return $firstChar . mb_substr($string, 1, null, $encoding);
     }
+
+    public static function array_find(array $array, callable $callback)
+    {
+        foreach ($array as $key => $value) {
+            if ($callback($value, $key)) {
```

Comment:
```suggestion
            if (true === $callback($value, $key)) {
```

i really wonder why not to rely on strict, boolean interface of callback result

Reply:
Thus typecheck would not match the nativ implementation which also has no typecheck: 

```
var_dump(array_find(['', 'aa'], fn($value) => strlen($value))); // string(2) "aa"
```